spiceaidocs/content/en/getting-started/postgres/supabase/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+type: docs
+title: "Getting Started with Supabase as a Data Accelerator"
+linkTitle: "Supabase Data Accelerator"
+description: 'Configure a Supabase PostgreSQL instance as a Data Accelerator'
+weight: 60
+---
+
+### Follow these steps to get started with Supabase as a Data Accelerator.
+
+**Step 1.** Create a Supabase account and project [https://supabase.com/](https://supabase.com/). Supabase will set up a new database as part of the account/project creation. Refer to the [Supabase Documentation](https://supabase.com/docs/guides/database/overview) for creating new databases in an existing project.
+
+**Step 2.** Open the newly created Supabase project and select `Project Settings` from the sidebar navigation.
+
+**Step 3.** Navigate to `Database` under `Configuration`.
+
+**Step 4.** Find the `Connection parameters` section.
+![Connection parameters](https://imagedelivery.net/HyTs22ttunfIlvyd6vumhQ/02a30c41-072c-49cc-cba3-e29f35ca6800/public)
+
+**Step 5.** Start the Spice.ai runtime with the following dataset loaded, copying the connection parameters from Supabase:
+
+```
+- from: spice.ai/eth.recent_blocks
+  name: eth_recent_blocks
+  acceleration:
+    refresh_mode: full
+    refresh_interval: 10s
+    engine: postgres
+    params:
+      pg_host: [Host]
+      pg_db: [Database name]
+      pg_port: "[Port]"
+      pg_user: [User]
+      pg_pass: [Password]
```

Comment:
Let's use the secret store for the password.

Reply:
Since this is a Postgres quickstart to get users up and running as fast as possible I think it should contain as little extra knowledge as possible. Maybe we should have a separate secrets quickstart instead?

Reply:
Fair, but let's at least mention the ability to use secrets with a link